packages/store/src/lib/resource-config.test.ts

```diff
@@ -43,19 +44,19 @@ describe('parseResourceConfigFlags', () => {
         },
         customers: {
           identifier: {
-            field: 'EMAIL',
+            field: 'EMAIL_ADDRESS',
             customId: undefined,
           },
         },
       })
     })
 
     test('overwrites identifier when same resource appears multiple times', () => {
-      const result = parseResourceConfigFlags(['products:handle', 'products:title'])
```

Comment:
when was title supported as an identifier? I dont see it [here](https://github.com/shop/world/blob/main/areas/core/shopify/components/platform/bulk_data_transfer/app/models/graph_api/admin/bulk_data_resource_config_input.rb)

Reply:
deleting all the tests with customers or products:title since they arent supported for EA/GA